config/prod/rna-seq-reprocessing-instance-role-v001.yaml

```diff
@@ -0,0 +1,4 @@
+template_path: remote/toil-cluster-role-and-policy.yaml
```

Comment:
remove "remote" since "toil-cluster-role-and-policy.yaml" is in the root of the templates folder